gatsby-browser.js

```diff
@@ -1,2 +1,7 @@
-/* eslint-env node */
-require('./src/styles/global.css')
+import React from 'react'
+
+import ModesProvider from './src/components/organisms/SwitchableMode/Provider'
+
+export const wrapRootElement = ({ element }) => (
+  <ModesProvider>{element}</ModesProvider>
+)
```

Comment:
I think it'd be better to just wrap our components in the normal React area in this rather than use the `wrapRootElement` escape hatch.